website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step09-add-prometheus-grafana.sh

```diff
@@ -3,8 +3,9 @@ helm repo add prometheus-community https://prometheus-community.github.io/helm-c
 helm repo update
 
 kubectl create namespace monitoring
+export KARPENTER_NAMESPACE=${KARPENTER_NAMESPACE:-kube-system}
```

Comment:
Everything else looks good, just one nit. We already export the `KARPENTER_NAMESPACE` in the beginning of the getting started guide. Is it necessary to export it again here? https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/#2-set-environment-variables

Reply:
Thanks - I decided to remove the export as we are not doing the same for the KARPENTER_VERSION. My assumption was if that readers do not follow the tutorial in one go (and revisit this optional part later) readers might forget to export the environment variables.